activestorage/db/update_migrate/20191206030411_create_active_storage_variant_records.rb

```diff
@@ -1,11 +1,23 @@
 class CreateActiveStorageVariantRecords < ActiveRecord::Migration[6.0]
   def change
-    create_table :active_storage_variant_records do |t|
-      t.belongs_to :blob, null: false, index: false
+    # Use Active Record's configured type for primary and foreign keys
+    primary_key_type, foreign_key_type = primary_and_foreign_key_types
+
+    create_table :active_storage_variant_records, id: primary_key_type do |t|
+      t.belongs_to :blob, null: false, index: false, type: foreign_key_type
```

Comment:
I think this reference column should follow the type of the existing `id` column on the `active_storage_blobs` table.

Reply:
@kamipo Ah, sure! Changed.